op-scim-bridge/chart/op-scim-bridge-mp/templates/deployer-cleanup.yaml

```diff
@@ -7,16 +41,20 @@ metadata:
     app.kubernetes.io/name: {{ .Values.name }}
     app.kubernetes.io/component: cleanup
 spec:
-  schedule: "*/1 * * * *" # once a minute
-  successfulJobsHistoryLimit: 0
+  schedule: "*/10 * * * *" # every 10 minutes
```

Comment:
TY ❤️

Reply:
I'm glad you like it. My thought here is that we need not run it _that_ frequently since for most of it's life this job will do nothing. Decreasing the frequency also gives us more time in-between for timeouts to occur and for us to inspect the logs.

A big contributor to the problem we saw was that we would start a cleanup job every minute, wait up to a minute before we tried to delete anything and each job would restart up to 3 times when it ran into an error. The new config is much more conservative, and also puts measures in place to avoid unnecessary parallel execution and retries.

Reply:
I had initially set it to 1 minute as I wanted the user to have the ability to redeploy on the same cluster as soon as possible, but honestly, I don't know that anyone's done that. So I think this is fine. 👍🏻